--- scsidev-2.35/Makefile.in	2002-10-03 15:43:08.000000000 +0000
+++ scsidev-2.35.new/Makefile.in	2005-01-03 23:15:37.767389208 +0000
@@ -48,8 +48,8 @@
 	cd $(srcdir) && autoconf
 
 install: scsidev
-	install -d $(bindir)
-	install -m 755 -s scsidev $(bindir)
+	install -d $(DESTDIR)$(bindir)
+	install -m 755 -s scsidev $(DESTDIR)$(bindir)
 	install -d $(mandir)
 	install -m 644 scsidev.8 $(mandir)
 	gzip -9f $(mandir)/scsidev.8
